composer.json

```diff
@@ -4,21 +4,24 @@
   "minimum-stability": "stable",
   "prefer-stable": true,
   "require": {
-    "php": ">=8.0",
+    "php": ">=8.1",
     "ext-ctype": "*",
     "ext-iconv": "*",
     "doctrine/doctrine-bundle": "^2.8",
     "doctrine/doctrine-migrations-bundle": "^3.2",
     "doctrine/orm": "^2.14",
     "easycorp/easyadmin-bundle": "^4.5",
     "nelmio/security-bundle": "^3.0",
+    "surfnet/stepup-saml-bundle": "dev-feature/support-sf6-dependencies",
     "symfony/console": "6.2.*",
     "symfony/dotenv": "6.2.*",
     "symfony/flex": "^2",
     "symfony/framework-bundle": "6.2.*",
+    "symfony/http-foundation": "6.2.*",
+    "symfony/monolog-bundle": "^3.0",
     "symfony/runtime": "6.2.*",
-    "symfony/yaml": "6.2.*",
-    "symfony/monolog-bundle": "^3.0"
+    "symfony/webpack-encore-bundle": "^1.16",
```

Comment:
Denk dat je al wel doorhad dat 1 MR voor de combinatie van webpack encore en federated login niet ideaal is.... :)